src/dmap_common.h

```diff
@@ -6,6 +6,7 @@
 #include <event2/http.h>
 
 #include "db.h"
+#include "httpd.h"
```

Comment:
Does not belong here, is not referenced in dmap_common.h. Should be in the .c - if it is required at all.